src/VisualStudio/NuGet.Packaging.VisualStudio/Templates/Projects/Package/Package.vstemplate

```diff
@@ -28,7 +28,7 @@
   </WizardExtension>
   <WizardData>
     <packages repository="extension" repositoryId="NuGet.Packaging">
-      <package id="NuGet.Build.Packaging" version="0.1.71-dev" />
+      <package id="NuGet.Build.Packaging" version="0.1.86-dev" />
```

Comment:
Updating these shouldn't be necessary anymore. The [build target updates them](https://github.com/NuGet/NuGet.Build.Packaging/blob/dev/src/VisualStudio/NuGet.Packaging.VisualStudio/NuGet.Packaging.VisualStudio.targets#L78-L81) before deploying the extension.